linkerd/admin/src/main/scala/io/buoyant/linkerd/admin/HttpIdentifierHandler.scala

```diff
@@ -0,0 +1,46 @@
+package io.buoyant.linkerd.admin
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.fasterxml.jackson.module.scala.DefaultScalaModule
+import com.fasterxml.jackson.module.scala.experimental.ScalaObjectMapper
+import com.twitter.finagle.Service
+import com.twitter.finagle.http._
+import com.twitter.io.Buf
+import com.twitter.util.{Future, Return, Throw, Try}
+import io.buoyant.router.RoutingFactory
+import io.buoyant.router.RoutingFactory.{IdentifiedRequest, UnidentifiedRequest}
+
+case class IdResult(path: Option[String] = None, dtab: Option[String] = None, error: Option[String] = None)
+
+object Json {
+  private[this] val mapper = new ObjectMapper with ScalaObjectMapper
```

Comment:
TIOLI: I think there's a setting that makes Jackson not serialize fields with null values.  It might be nice to omit the "error" field if there's no error.